src/routes/index.jsx

```diff
@@ -25,16 +28,20 @@ const Routes = () => (
                 <Route path="/forgot-password" component={PasswordRequestContainer} />
                 <Route path="/reset-password/:token" component={PasswordResetContainer} />
                 <Layout>
-                    <Route path="/" exact component={HomeView} />
+                    <ToastContainer />
                     <Switch>
+                        <Route path="/" exact component={HomeView} />
+                        <Route path="/articles/:slug" exact component={Article} />
                         <Route path="/articles" exact component={ArticlesView} />
                         <Route path="/articles/create" exact component={CreateArticle} />
                         <Route path="/articles/update/:slug" exact component={UpdateArticle} />
-                        <Route path="/articles/:slug" component={Article} />
+                        <Route path="/tags" component={LoginView} />
+                        <Route exact path="/:username" component={UserProfileContainer} />
+                        <Route exact path="/:username/edit" component={EditProfileContainer} />
+                        <Route exact path="/:username/followers" component={FollowersFollowing} />
+                        <Route exact path="/:username/following" component={FollowersFollowing} />
+                        <Route path="/email-verification/:token" exact component={EmailVerificationView} />
                     </Switch>
```

Comment:
@oma0256, this is nice work. the UI looks good. I think the switch for the URLs is not working as desired as I am now not able to get to the create articles page. Instead, it tries to look for article with slug: articles and returns NOT FOUND. Would you look into that please

Reply:
@pbkabali Thanks for bringing this to my attention let me look into this